packages/create-svelte/bin.js

```diff
@@ -21,16 +21,41 @@ async function main() {
 	console.log(gray(`\ncreate-svelte version ${version}`));
 	console.log(disclaimer);
 
-	let cwd = process.argv[2] || '.';
-
-	if (cwd === '.') {
-		const opts = await prompts([
+	const package_name = await prompts(
+		[
 			{
 				type: 'text',
-				name: 'dir',
-				message: 'Where should we create your project?\n  (leave blank to use current directory)'
+				name: 'packagename',
+				message: 'What would you like to name your project?',
+				validate: (value) => {
+					const regex = new RegExp(/^(@[a-z0-9-~][a-z0-9-._~]*\/)?[a-z0-9-~][a-z0-9-._~]*$/);
+					const match = regex.test(value);
+					return !match
+						? 'Value is not a valid NPM package name. Package names must be lowercase and one word, and may contain hyphens and underscores, or be scoped and of the form @scope/package.'
```

Comment:
isn't `match` already boolean value? Also I don't know if You want to put error message here. (When you are not throwing it there)...

`return match ?? 'Value is not...'`

Reply:
Ah, you're sort of right. The error message would never be returned with `??` (nullish coalescing) -- but I should be using `matched || "Blah blah blah"`. I'll fix it later.